extended/src/test/java/apoc/ml/OpenAIOpenLMIT.java

```diff
@@ -31,12 +34,169 @@ public class OpenAIOpenLMIT {
+        );
+    }
+
+@Test
+public void completionWithAnthropicSmallTokenSize() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 1
+    );
+
+    testCall(db, COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            (row) -> {
+                var result = (Map<String,Object>) row.get("value");
+                var contentList = (List<Map<String, Object>>) result.get("content");
+                Map<String, Object> content = contentList.get(0);
+                String generatedText = (String) content.get("text");
+                String[] wordCount = generatedText.toLowerCase().split(" ");
+                assertEquals(1, wordCount.length);
+            });
+}
+
+@Test
+public void completionWithAnthropicCustomVersion() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 1024,
+            ANTHROPIC_VERSION, "2023-06-01"
+    );
+    testCall(db, COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            (row) -> {
+                var result = (Map<String,Object>) row.get("value");
+                var contentList = (List<Map<String, Object>>) result.get("content");
+                Map<String, Object> content = contentList.get(0);
+                String generatedText = (String) content.get("text");
+                assertTrue(generatedText.toLowerCase().contains("blue"),
+                        "Actual generatedText is " + generatedText);
+            });
+}
+
+@Test
+public void completionWithAnthropicWrongVersion() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 1024,
+            ANTHROPIC_VERSION, "ajeje"
+    );
+
+    assertFails(
+            db,
+            COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            "Server returned HTTP response code: 400 for URL: https://api.anthropic.com/v1/messages"
+    );
+}
+
+@Ignore
```

Comment:
Ripensandoci, toglilo proprio questo, metti solo nella documentazione qualcosa tipo:

```
NOTE: currently https://docs.anthropic.com/en/api/messages-streaming[`stream: true`] is not supported, since the result of apoc.ml.openai must be in JSON format
```